src/vehicle.cpp

```diff
@@ -875,6 +875,13 @@ void vehicle::use_controls()
         menu.addentry( release_remote_control, true, 'l', _("Stop controlling") );
     }
 
+    bool has_electronic_controls = false;
+
+    for( size_t p = 0; p < parts.size(); p++ ) {
```

Comment:
I realized that's better to define the relative coordinate of selected tile instead of ID of one exact part (what if you will install two parts with "control" flag, and only one of them will have a flag for electronic controls? I guess it's not possible now but it's easy to implement such functionality).